Makefile

```diff
@@ -24,15 +25,19 @@ bootstrap:
 
 .PHONY: test-integration
 test-integration:
-	go test ./tests/... ${TEST_OPTS}
+	DEFAULT_EVENTUALLY_TIMEOUT="30s" ginkgo $TEST_OPTS tests/
```

Comment:
so ive made enough changes to the tests that I had significant failures with a 10s timeout. I am weary to move it that low given the failures i was seeing.